frb_dart/lib/src/basic.dart

```diff
@@ -64,29 +74,43 @@ abstract class FlutterRustBridgeBase<T extends FlutterRustBridgeWireBase> {
 
   /// Similar to [executeNormal], except that this will return a [Stream] instead of a [Future].
   @protected
-  Stream<S> executeStream<S>(FlutterRustBridgeTask<S> task) async* {
+  Stream<S> executeStream<S, E extends Object, P extends Object>(
+      FlutterRustBridgeTask<S, E, P> task) async* {
     final receivePort = ReceivePort();
     task.callFfi(receivePort.sendPort.nativePort);
 
     await for (final raw in receivePort) {
       try {
-        yield _transformRust2DartMessage(raw, task.parseSuccessData);
+        yield _transformRust2DartMessage(raw, task.parseSuccessData,
+            task.parseErrorData, wire2apiPanicError);
       } on _CloseStreamException {
         receivePort.close();
       }
     }
   }
 
-  S _transformRust2DartMessage<S>(
-      dynamic raw, S Function(dynamic) parseSuccessData) {
+  S _transformRust2DartMessage<S, E extends Object>(
+      dynamic raw,
+      S Function(dynamic) parseSuccessData,
+      E Function(dynamic)? parseErrorData,
+      E Function(dynamic)? parsePanicData) {
```

Comment:
Guess this is wrong? `parsePanicData` always return PanicError, while `E` can be anyhow error, custom error types, etc

Shall it be `PanicError` instead of `E`